tests/rewards/test_reward_nets.py

```diff
@@ -182,60 +182,54 @@ def forward(self):
     assert net.device == th.device("cpu")
 
 
-@pytest.mark.parametrize("reward_net_cls", [reward_nets.BasicRewardNet])
-@pytest.mark.parametrize("normalize_input_layer", [networks.RunningNorm])
-@pytest.mark.parametrize("normalize_output_layer", [networks.RunningNorm])
-def test_training_regression(
-    reward_net_cls,
-    normalize_input_layer,
-    normalize_output_layer,
-):
+@pytest.mark.parametrize("normalize_input_layer", [None, networks.RunningNorm])
+def test_training_regression(normalize_input_layer):
     """Test reward_net normalization by training a regression model."""
-    # TODO(yawen):
     venv = DummyVecEnv([lambda: gym.make("CartPole-v0")] * 2)
-    reward_net = reward_net_cls(
+    reward_net = reward_nets.BasicRewardNet(
         venv.observation_space,
         venv.action_space,
         normalize_input_layer=normalize_input_layer,
     )
-    if normalize_output_layer is not None:
-        reward_net = reward_nets.NormalizedRewardNet(
-            reward_net,
-            normalize_output_layer,
-        )
+    norm_rew_net = reward_nets.NormalizedRewardNet(
+        reward_net,
+        normalize_output_layer=networks.RunningNorm,
+    )
 
     # Construct a loss function and an Optimizer.
     criterion = th.nn.MSELoss(reduction="sum")
-    optimizer = th.optim.SGD(reward_net.parameters(), lr=1e-6)
+    optimizer = th.optim.SGD(norm_rew_net.parameters(), lr=1e-6)
 
-    for _ in range(20):
-        # Forward pass: Compute predicted y by passing x to the model
-        random = base.RandomPolicy(venv.observation_space, venv.action_space)
+    # Getting transitions from a random policy
+    random = base.RandomPolicy(venv.observation_space, venv.action_space)
+    for i in range(2):
```

Comment:
```suggestion
    for _ in range(2):
```
i is not used AFAICT